HerPublicWebsite.BusinessLogic/Models/OsPlacesResult.cs

```diff
@@ -0,0 +1,98 @@
+using Newtonsoft.Json;
+
+public record class OsPlacesResult
```

Comment:
If this is going to be the data that comes from the OS Places API then please end the name with Dto for consistency.
If this is going to be the class that the API code passes to the rest of the business logic then it won't need a lot of these fields - just the address line, UPRN, and custodian code I think.